### PR TITLE
gh-105481: expose opcode metadata via the _opcode module

### PR DESCRIPTION
**Pull request by iritkatriel**
* Issue: gh-105481

### CODE REVIEW

**Comment by iritkatriel** [comment]
There's a problem here with the definition of the metadata table. I'll try to sort it out tomorrow.

**Review by gvanrossum — APPROVED** [review]

File: Misc/NEWS.d/next/Library/2023-07-13-16-04-15.gh-issue-105481.pYSwMj.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Expose opcode metadat though :mod:`_opcode`.
```
Comment (iritkatriel) [inline Misc/NEWS.d/next/Library/2023-07-13-16-04-15.gh-issue-105481.pYSwMj.rst]:
```suggestion
Expose opcode metadata through :mod:`_opcode`.
```

**Review by gvanrossum — APPROVED** [review]
Something worries me about the amount of repetition for each flag. But the alternative (exposing the flags as an int, and the constants) seems worse. So go for it!

**Comment by iritkatriel** [comment]
Yes, I didn't want to expose the bitmap implementation.

**Pull request merged**